source/docs/being-a-team/how-to-ask-effective-questions.rst

```diff
@@ -0,0 +1,69 @@
+.. include:: <isonum.txt>
+
+How to Ask Effective Questions
+==============================
+
+Asking questions effectively is one of the most important skills to develop, especially for your life outside of robotics.
+Being able to learn from your questions properly is crucial and as such, you should be able to ask good questions, help those who want to help you, and make the most of what you are taught.
+
+Research First
+--------------
+
+Before you ask a question, you should be doing your own research to find the answer.
+This will take more time, it will take more effort, but it is a crucial skill to learn to find solutions on your own, and learn how to research effectively to find solutions to your problems.
+In the end, this research will pay off in the long run, helping you to learn about topics relevant to your question, and often helping you to understand the roots of the problems you encounter.
+
+Here are some steps that you could take to research your problem before asking someone a question about your current problem.
+
+- **Google** - Simply search your problem.
+  You will be surprised by how often this can quickly solve your problem although you may not have expected it to.
+  Try word the question in many different ways to find similar questions that may have already been asked.
+
+.. tip::
+    Add "filetype:pdf" to the beginning of your search to filter the search results into only .pdf documents.
+    This can help you when looking for past documents or sections of textbooks.
+
+- **Youtube** - Just like google, there are many great resources on Youtube which can help with very specific, obscure problems.
+- **Reddit and Discord** - If you can't find your own question online, the FTC Reddit and Discord will be the other most likely place to have a past question similar to your own.
+  Try searching through the past questions with words related to your question and see if a similar question has been asiked before.
+  You may find information in this discussion which can help you to solve your own problem.
+- **Past Examples** - Try find past implementations of whatever problem you are trying to solve.
```

Comment:
```suggestion
- **Past Examples** - For programming questions, it can sometimes be helpful to find past implementations of whatever problem you are trying to solve through [GitHub Code Search](https://github.com/search?type=code&auto_enroll=true).
```

Reply:
This could be used for mech stuff too, but not really sure where you'd find things in a similar way to code search.

Reply:
Extracted out into a tip to make it more clear.